src/pudl/etl/cli.py

```diff
@@ -145,7 +145,13 @@ def pudl_etl(
             },
         },
     }
-    run_config.update(get_dagster_execution_config(dagster_workers))
+
+    run_config.update(
+        get_dagster_execution_config(
+            num_workers=dagster_workers,
+            tag_concurrency_limits=pudl.etl.default_tag_concurrency_limits,
```

Comment:
Should I add EPA CEMS concurrency value as a command line arg with a default value of 2 for `pudl_etl`?

Reply:
Would we want to run it with higher concurrency in the nightly builds because the machine has so much memory? I have no trouble on my laptop with 32GB and 2 concurrent processes.

Although, having several ways to configure the same parameters can get confusing and make it harder to debug when something goes wrong.  As it is, it's hard-coded to 2, unless someone edits it in the UI right? How does Dagster typically expect people to configure this kind of thing?

Reply:
I agree tt would be nice to be able to change concurrency for CEMS from the command line, given the GCP machines have a bunch of memory. I'll add it as an arg.

Reply:
We could just move towards using the dagster CLI commands to execute jobs specified in `pudl.etl` instead of using `pudl_etl` and `ferc_to_sqlite` and reconstructing the run configs.

```
dagster job execute -m pudl.etl -j [etl_fast|etl_fast_no_cems|etl_full|etl_full_no_cems]

dagster job execute -m pudl.etl -j [ferc_to_sqlite_fast|ferc_to_sqlite_full]
```

The PUDL commands currently use `dagster.build_reconstructable_job()` which is required to execute multiproccess jobs from python. The function is still experimental and not super straightforward. We kept `pudl_etl` and `ferc_to_sqlite` around as people migrated to executing from the UI. Seems like we're at the point where we are mostly executing jobs from the UI and just using the CLI commands for CI.

Reply:
Also, if we use the dagster CLI we won't have to translate click arguments into dagster config.

Reply:
If there's a clean and well supported way to run the things that we run using dagster directly it seems like that's probably the direction to move in.

Reply:
Sounds good 👍 I'll create an issue to replace the commands